Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -564,6 +564,21 @@ public ElasticsManager ElasticsManager
         private Vector3[] boundsCorners = new Vector3[8];
         public Vector3[] BoundsCorners { get; private set; }
 
+        // Current actual flattening axis, derived from FlattenAuto, if set
+        private FlattenModeType ActualFlattenAxis {
```

Comment:
Question: Why "Actual"FlattenAxis?

Reply:
Answer: `flattenAxis` can be `FlattenAuto`, which is not actually an axis. `FlattenAuto` resolves to either X, Y, or Z depending on the particular dimensions of the target bounds. The per-axis handles require an _actual_ axis to flatten on, not an "auto" axis, so now `BoundsControl` "resolves" the flatten axis into an actual, real axis. Hence "actual" flatten axis :)